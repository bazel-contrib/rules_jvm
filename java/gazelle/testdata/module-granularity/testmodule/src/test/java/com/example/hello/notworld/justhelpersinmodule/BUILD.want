load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

java_test_suite(
    name = "justhelpers-tests",
    srcs = [
        "Helper.java",
        "withdirectory/Helper.java",
    ],
    _gazelle_imports = [
        "com.example.hello.notworld.justhelpers",
        "com.example.hello.notworld.justhelpers.withdirectory",
        "java.lang.String",
    ],
    _java_packages = [
        "com.example.hello.notworld.justhelpers",
        "com.example.hello.notworld.justhelpers.withdirectory",
    ],
    runner = "junit5",
    runtime_deps = [
        "@maven//:org_junit_jupiter_junit_jupiter_engine",
        "@maven//:org_junit_platform_junit_platform_launcher",
        "@maven//:org_junit_platform_junit_platform_reporting",
    ],
)
