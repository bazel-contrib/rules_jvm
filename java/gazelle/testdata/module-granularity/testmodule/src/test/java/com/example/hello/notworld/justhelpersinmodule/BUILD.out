load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

# gazelle:java_module_granularity module

java_test_suite(
    name = "justhelpersinmodule-tests",
    srcs = [
        "Helper.java",
        "withdirectory/Helper.java",
    ],
)
